impeller/golden_tests/screenshot.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_IMPELLER_GOLDEN_TESTS_SCREENSHOT_H_
+#define FLUTTER_IMPELLER_GOLDEN_TESTS_SCREENSHOT_H_
+
+#include <cstddef>
+#include <cstdint>
+#include <memory>
+#include <string>
+
+namespace impeller {
+namespace testing {
+
+class Screenshot {
+ public:
+  virtual ~Screenshot() = default;
+
+  virtual const uint8_t* GetBytes() const = 0;
```

Comment:
Now that we have this neatly seperated it might be nice to ad 1-line comments at least for the non-obvious APIs (like I'm not sure what GetBytesPerRow() does for example, though maybe it's obvious to graphics engineers?)

Reply:
Done, added docstrings.